packages/SwingSet/src/kernel/vatAdmin/vatAdminWrapper.js

```diff
@@ -8,16 +8,23 @@
 import harden from '@agoric/harden';
 import { producePromise } from '@agoric/produce-promise';
 
+function producePRR() {
+  const { promise, resolve, reject } = producePromise();
+  return [promise, { resolve, reject }];
+}
+
 export default function setup(syscall, state, helpers) {
   function build(E, D) {
-    const pending = new Map();
+    const pending = new Map(); // vatID -> { resolve, reject } for promise
```

Comment:
People keep asking me why I'm not using makeStore() instead of Map. Is there a good reason not to switch in the kernel?

Reply:
no better reason than inertia, I guess

We aren't doing anything fancy with our Map: the key comes from elsewhere (which promises to make them unique), and there isn't enough code here to benefit from being able to detach e.g. `pending.get` in a functional-programming style. I can try to reach for Store instead of Map next time, but I'm not sure this instance is worth the effort (or the extra time for some reader down the line to say "huh, what's a Store?").